orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartWaitingExecutionsHandler.kt

```diff
@@ -51,11 +51,10 @@ class StartWaitingExecutionsHandler(
             ?.let { newest ->
               log.info("Starting queued {} {} {}", newest.application, newest.name, newest.id)
               queue.push(StartExecution(newest))
-              (pipelines - newest)
-                .also { queued ->
-                  log.info("Dropping queued {} {} {}", newest.application, newest.name, queued.map { it.id })
-                }
+              pipelines
+                .filter { it.id != newest.id }
```

Comment:
This is probably a cleaner way of filtering anyway.

Reply:
👍 I think this is a lot more obvious about what the filter is actually doing.

Reply:
Yeah, I still need to get the test to fail when doing it the old way otherwise we can't trust it to catch any future regressions.